.github/workflows/db-ops.yaml

```
@@ -0,0 +1,206 @@
+
+      - name: Run ArgoCD Action
+        id: argocd_api
+        uses: ./.github/actions/argocd-action
+        with:
+          argocd_username: ${{ secrets.ARGOCD_USERNAME }}
+          argocd_password: ${{ secrets.ARGOCD_PASSWORD }}
+          argocd_server: ${{ secrets.ARGOCD_SERVER }}
+        
+  send-to-slack:
+    runs-on: ubuntu-latest
+    needs: [update-helm-chart,build-and-push-ee-image]
```

yamllint

[warning] too few spaces after comma

(commas)